Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -254,17 +254,9 @@ void FSpatialGDKEditorToolbarModule::GenerateInteropCodeButtonClicked()
 	ShowTaskStartNotification("Generating Interop Code");
 	bInteropCodeGenRunning = true;
 
-	ClassHeaderMap InteropGeneratedClasses;
-	if (!GenerateClassHeaderMap(InteropGeneratedClasses))  // Checks that all classes are found and generate the class mapping.
+	TFunction<bool()> CodegenTask = []()
```

Comment:
Do we need a lambda here? Can't we just pass `SpatialGDKGenerateInteropCode` to be implicitly converted to a `TFunction`?